server/php/UploadHandler.php

```diff
@@ -1057,7 +1085,7 @@ protected function handle_file_upload($uploaded_file, $name, $size, $type, $erro
         $file = new \stdClass();
         $file->name = $this->get_file_name($uploaded_file, $name, $size, $type, $error,
             $index, $content_range);
-        $file->size = $this->fix_integer_overflow((int)$size);
+        $file->size = strval($size);
```

Comment:
Again here, with setting `$file->size` to a string, can you confirm that the comparison [here](https://github.com/servocoder/jQuery-File-Upload/blob/master/server/php/UploadHandler.php#L1098) will still work in all cases?